README.md

```diff
@@ -15,7 +15,7 @@ To inline HTML without getting remote resources, using default options:
 
 ```js
 var juice = require('juice');
-var result = juice("<style>div{color:red;}</style><div/>");
+var result = juice("<style>div{color:red;}</style><div><div/>");
```

Comment:
Is this supposed to be `<div></div>` instead? I'm not sure where you got an extra unclosed `div` from